api/src/main/java/org/pentaho/platform/api/genericfile/IGenericFileProvider.java

```diff
@@ -149,4 +154,6 @@ IGenericFileTree getTree( @NonNull GetTreeOptions options )
    * @throws OperationFailedException If the operation fails for some other (checked) reason.
    */
   boolean hasAccess( @NonNull GenericFilePath path, @NonNull EnumSet<GenericFilePermission> permissions );
+
+  IGenericFileContentWrapper getFileContentWrapper(@NonNull String path ) throws OperationFailedException, IOException;
```

Comment:
seems in consistent that every method in the interfaces has "`GenericFilePath path`" but then this new method has a type of `String`

Reply:
Good point. I will fix this...